.circleci/config.yml

```diff
@@ -234,43 +234,52 @@ jobs:
            install-and-test-ext --target check-citus-upgrade-mixed --citus-pre-tar /install-pg11-citusv8.3.0.tar
           no_output_timeout: 2m
 
-
+  check-merge-to-enterprise:
+    docker:
+      - image: buildpack-deps:stretch
+    working_directory: /home/circleci/project  
+    steps:
+      - checkout
+      - run:
+          command: |
+            sh ./src/test/scripts/check_enterprise_merge.sh
 
 
 
 workflows:
   version: 2
   build_and_test:
     jobs:
-      - build
-      - check-style
-      - check-sql-snapshots
+      - check-merge-to-enterprise
+      # - build
+      # - check-style
```

Comment:
I will enable the tests after we are okay with the other parts. ( I disabled them to not cause unnecessary run of other jobs since I push to this branch a lot)